src/main/java/org/osc/controller/nsc/utils/RedirectionApiUtils.java

```diff
@@ -201,10 +201,40 @@ public InspectionPortEntity txInspectionPortEntityById(String id) {
         return this.em.find(InspectionPortEntity.class, id);
     }
 
+    public List<InspectionHookEntity> txInspectionHookEntities(String inspectionPort) {
```

Comment:
I dont think this will be necessary if you dont make the inspectionport the parent.

Reply:
ok